rmw_cyclonedds_cpp/src/rmw_node.cpp

```diff
@@ -1499,6 +1499,13 @@ extern "C" rmw_ret_t rmw_context_fini(rmw_context_t * context)
   return ret;
 }
 
+extern "C" rmw_ret_t rmw_notify_participant_dynamic_network_interface(rmw_context_t * context)
+{
+  static_cast<void>(context);
+  // RMW_SET_ERROR_MSG("rmw_notify_participant_dynamic_network_interface not implemented for rmw_cyclonedds_cpp");
```

Comment:
why the error message is commented?

Reply:
Ok perfect.
PR looks good to me, feel free to merge as soon as your tests look good.